scipy/stats/tests/test_distributions.py

```diff
@@ -3699,6 +3701,30 @@ def test_pdf(self):
         dist = stats.dgamma(a)
         assert_equal(dist.pdf(x), res)
 
+    # mpmath was used to compute the expected values.
+    # For x < 0, cdf(x, a) is mp.gammainc(a, -x, mp.inf, regularized=True)/2
+    # For x > 0, cdf(x, a) is (1 + mp.gammainc(a, 0, x, regularized=True))/2
+    # E.g.
+    #    from mpmath import mp
+    #    mp.dps = 50
+    #    print(float(mp.gammainc(1, 20, mp.inf, regularized=True)/2))
+    # prints
+    #    1.030576811219279e-09
+    @pytest.mark.parametrize('x, a, expected',
+                             [(-20, 1, 1.030576811219279e-09),
+                              (-40, 1, 2.1241771276457944e-18),
+                              (-50, 5, 2.7248509914602648e-17),
+                              (-25, 0.125, 5.333071920958156e-14)])
+    def test_cdf_ppf_sf_isf_tail(self, x, a, expected):
```

Comment:
This is only checking in the left tail. Do we think that the generic tests are adequate to cover the right tail?

Reply:
`check_cdf_ppf` checks roundtrip error at `p = [0.001, 0.5, 0.999]`, so it will check one value in each tail. The story is similar for checking sf/isf roundtrip and cdf + sf = 1. This confirms that the functions here are correctly related to one another in both tails, but it does link these to the distribution PDF.

`check_pdf` only compares that the PDF is the derivative of the CDF at the _median_. 

So, since the implementation branches depending on the tail, and since the generic tests don't check the relationship between CDF and PDF in each tail, what do you think we should do? Add one test at a moderate value in the other tail? (We will change the generic test eventually).